### PR TITLE
fix typo on the "whats new" page

### PR DESCRIPTION
**Pull request by VladBrok**
Fixes #SOME_ISSUE_NUMBER

Changes proposed in this pull request:
fix typo

-
-
-

### CODE REVIEW

File: site/en/docs/extensions/whatsnew/index.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: 'layouts/doc-post.njk'
 title: What's new in Chrome extensions
 description: 'Recent changes to the Chrome extensions platform, documentation, and policy'
 date: 2021-02-25
-updated: 2023-05-17
+updated: 2023-05-25
```
Comment (oliverdunk) [inline site/en/docs/extensions/whatsnew/index.md:6]: @AmySteam, do we usually bump this for spelling fixes?

**Review by AmySteam — CHANGES_REQUESTED** [review]
@VladBrok Thanks for this.

We can't merge until the CLA is signed and we have to wait until next week.

Thanks!

File: site/en/docs/extensions/whatsnew/index.md (hunk 1 above)
Comment (AmySteam) [reply site/en/docs/extensions/whatsnew/index.md:6]: I don't think so

**Comment by AmySteam** [comment]
Closing due to inactivity for 2 weeks.

**Pull request closed without merging**